Snakefile

```diff
@@ -293,6 +296,24 @@ if not config["enable"].get("build_natura_raster", False):
             shutil.copyfile(input[0], output[0])
 
 
+if config["enable"].get("retrieve_cost_data", True):
```

Comment:
This would fail with the default config file: the costs.csv file shall may be provided also by the databundles.
For this to work, thre is the need to remove in config/bundle_config the "/costs.csv" output.
Have you tried to run a default run locally also?